# -*- coding: utf-8 -*-
# Generated by Django 1.11.4 on 2018-09-15 09:38
from __future__ import unicode_literals

import django.contrib.auth.validators
from django.db import migrations, models


class Migration(migrations.Migration):

    dependencies = [
        ('main', '0022_merge_20180915_0936'),
    ]

    operations = [
        migrations.AddField(
            model_name='accomodation',
            name='mf',
            field=models.CharField(default='male', max_length=10),
        ),
        migrations.AlterField(
            model_name='customuser',
            name='username',
            field=models.CharField(error_messages={'unique': 'A user with that username already exists.'}, help_text='Required. 150 characters or fewer. Letters, digits and @/./+/-/_ only.', max_length=150, unique=True, validators=[django.contrib.auth.validators.ASCIIUsernameValidator()], verbose_name='username'),
        ),
        migrations.AlterField(
            model_name='regplayer',
            name='notes',
            field=models.CharField(blank=True, max_length=10000, null=True),
        ),
        migrations.AlterField(
            model_name='regplayer',
            name='uid',
            field=models.CharField(default='18CB0000', max_length=100),
        ),
    ]
